src/js/media-chrome-range.js

```diff
@@ -209,6 +211,8 @@ class MediaChromeRange extends window.HTMLElement {
     this.container = this.shadowRoot.querySelector('#container');
     this.range = this.shadowRoot.querySelector('#range');
     this.range.addEventListener('input', this.updateBar.bind(this));
+
+    this.#thumbWidth = parseInt(getComputedStyle(this).getPropertyValue('--media-range-thumb-width') || 10);
```

Comment:
we discussed in chat that this would not be changeable after construction.

I think for now this is okay because even when this was called in the `getBarColors()` method changing the CSS var would not have any effect if the media was not playing.

I'd rather have this optimization in than cover the rare case of someone changing the thumb width after initialization. there's also still the option to use CSS transforms, scale